source/bringauto/modules/ModuleHandler.cpp

```diff
@@ -46,32 +45,25 @@ void ModuleHandler::handleMessages() {
 		} else if(message.getMessage().has_devicestatus()) {
 			handleStatus(message.getMessage().devicestatus());
 		}
-
 		fromInternalQueue_->pop();
 	}
 }
 
 void ModuleHandler::checkTimeoutedMessages(){
 	for (const auto& [key, statusAggregator] : moduleLibrary_.statusAggregators) {
+		auto moduleLibraryHandler = moduleLibrary_.moduleLibraryHandlers.at(key);
 		if(statusAggregator->getTimeoutedMessageReady()){
-			struct buffer unique_devices {};
-			int ret = statusAggregator->get_unique_devices(&unique_devices);
+			std::list<structures::DeviceIdentification> unique_devices;
```

Comment:
add {}.
Add [] to all places which initialize C++/C structures pls.

Reply:
{} added everywhere